README.md

```diff
@@ -998,6 +998,8 @@ The same example with flags:
 However, keep in mind that Storers have to support regular expression and depending on the implementation of the storage handler the accepted syntax may vary.
 An error of `ErrNotImplemented` will be returned for those storage back-ends which do not support the `$regex` operator.
 
+The operator `$not` functions as an opposite operator to `$regex`
```

Comment:
We should stick to the MongoDB definition if implementing this:

https://docs.mongodb.com/manual/reference/operator/query/not/

Reply:
Did not see your comment in https://github.com/rs/rest-layer/issues/273#issuecomment-925864211 earlier. Maybe extend the text somewhat:

```suggestion
The operator `$not` functions as an opposite operator to `$regex`. Unlike MongoDB, we do not allow `$not` as a general negation operator.
```

Reply:
done